### PR TITLE
Update documentation TLS

### PR DESCRIPTION
**Pull request by kcadette**
Issue #, if available:

Description of changes:
Add TLS guidance to readme

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ FHIR Works on AWS is powered by single-function components. These functions prov
 
 - [Interface](https://github.com/awslabs/fhir-works-on-aws-interface) - Defines communication between the components.
 - [Routing](https://github.com/awslabs/fhir-works-on-aws-routing) - Accepts HTTP FHIR requests, routes it to the other components, logs the errors, transforms output to HTTP responses and generates the [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
+  - What is the recommended transport layer security (TLS) setting? 
+  
+    FHIR Works on AWS does not deploy a custom domain, and API Gateway does not allow FHIR Works on AWS to require TLS 1.2. Customers should configure FHIR Works on AWS to meet their internal security policies
```
Comment (ssvegaraju) [inline README.md]: Nit:
```suggestion
    FHIR Works on AWS does not deploy a custom domain, so API Gateway does not allow FHIR Works on AWS to require TLS 1.2. Customers should configure FHIR Works on AWS to meet their internal security policies.
```

**Review by ssvegaraju — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -41,6 +41,9 @@ FHIR Works on AWS is powered by single-function components. These functions prov
 
 - [Interface](https://github.com/awslabs/fhir-works-on-aws-interface) - Defines communication between the components.
 - [Routing](https://github.com/awslabs/fhir-works-on-aws-routing) - Accepts HTTP FHIR requests, routes it to the other components, logs the errors, transforms output to HTTP responses and generates the [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
+  - What is the recommended transport layer security (TLS) setting?
```
Comment (awsdrbalaji) [inline README.md:44]: Can we also talk about cipher suites that customers should avoid?

File: README.md [hunk 3]
```diff
@@ -41,6 +41,9 @@ FHIR Works on AWS is powered by single-function components. These functions prov
 
 - [Interface](https://github.com/awslabs/fhir-works-on-aws-interface) - Defines communication between the components.
 - [Routing](https://github.com/awslabs/fhir-works-on-aws-routing) - Accepts HTTP FHIR requests, routes it to the other components, logs the errors, transforms output to HTTP responses and generates the [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
+  - What is the recommended transport layer security (TLS) setting? 
+  
+    FHIR Works on AWS does not deploy a custom domain, so API Gateway does not allow FHIR Works on AWS to require TLS 1.2. Customers should configure FHIR Works on AWS to meet their internal security policies.
```
Comment (rsmayda) [inline README.md:46]: `to meet their internal security policies.` -> we should be more descriptive. What is AWS's advice? perhaps link to an AWS page

We should say something closer to "we advise x,y,z to secure your transport layer"

Comment (awsdrbalaji) [reply README.md:46]: This can be referenced: https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-custom-domain-tls-version.html

**Pull request merged**